.gitignore

```diff
@@ -2,3 +2,6 @@ node_modules
 keys.json
 package-lock.json
 lambdas/info.txt
+lambdas/file_access_patterns/out
```

Comment:
instead of mentioning specific out we can use the single expression for all the `out` directories, it will be better as later more patterns and nested directories could be added

Reply:
But we will be having only 3 access patterns..right?

Reply:
if later we have more than, in this case, we need to update .gitignore also why don't we use regex for a matching pattern to not include any out inside /lambdas directory.